rpc/src/lib.rs

```diff
@@ -103,6 +105,8 @@ extern crate rand_xorshift;
 #[cfg(test)]
 extern crate ethjson;
 #[cfg(test)]
+extern crate patricia_trie_ethereum as ethtrie;
+#[cfg(test)]
 extern crate transaction_pool as txpool;
```

Comment:
is `tx_pool` used only in tests?

Reply:
No, but `miner/src/lib.rs` imports its own. `private-tx` does the same.